boards/NUCLEOL476RG.py

```diff
@@ -50,7 +50,7 @@
   'usart' : 3,
   'spi' : 4,
   'i2c' : 3,
-  'adc' : 1,
+  'adc' : 2,
```

Comment:
Ahh - thanks - yes, this should be 3. I'll change it.

Sorry about breaking that bit of the build - I guess I didn't notice because of the other thing.

Now we know what the fix it I'll move the pyserial install into `provision.sh` though - then it's only installed for ESP32 where it's needed and the other builds stay a bit faster